include/fizzy/fizzy.h

```diff
@@ -275,6 +275,28 @@ uint8_t* fizzy_get_instance_memory_data(FizzyInstance* instance);
 /// @note    Function returns memory size regardless of whether memory is exported or not.
 size_t fizzy_get_instance_memory_size(FizzyInstance* instance);
 
+/// Find exported function by name.
+///
+/// @param  instance        Pointer to instance.
+/// @param  name            The function name. NULL-terminated string. Cannot be NULL.
+/// @param  out_function    Pointer to output struct to store the found function. Cannot be NULL.
+///                         If function is found, associated context is allocated, which must exist
+///                         as long as the function can be called by some other instance, and should
+///                         be destroyed with fizzy_free_exported_function afterwards.
+///                         When function is not found (false returned), this out_function is not
+///                         modified, and fizzy_free_exported_function must not be called.
+/// @returns                true if function was found, false otherwise.
+bool fizzy_find_exported_function(
+    FizzyInstance* instance, const char* name, FizzyExternalFunction* out_function);
+
+/// Free resources associated with exported function.
+///
+/// If passed pointer is NULL, has no effect.
```

Comment:
This is a bit confusing since we never return `FizzyExternalFunction` as a pointer.

Reply:
Well we pass it by non-const pointer in `fizzy_find_exported_function`

Reply:
I understand that. This comment is connected to the one above.